src/test/java/com/google/cloud/tools/maven/run/RunnerTest.java

```diff
@@ -120,25 +111,29 @@ public void testRunAsync() throws MojoExecutionException, IOException, AppEngine
   public void testProcessServices_singleService() throws MojoExecutionException {
     List<Path> userConfiguredServices = ImmutableList.of(STANDARD_PROJECT_WEBAPP);
     when(runMojo.getServices()).thenReturn(userConfiguredServices);
-    List<Path> processedServices = testRunner.processServices();
-    Assert.assertEquals(userConfiguredServices, processedServices);
+    testRunner.processServices();
+    // no exception pass
```

Comment:
Can we not preserve the `assertEquals` test? Seems meaningful to check the return value.

Reply:
Yeah, I went back and forth on that, this does less "processing" now. But I can change it back, makes sense.